CHANGES.md

```diff
@@ -9,6 +9,10 @@
   but is no longer officially supported by cloudpickle)
   ([issue #276](https://github.com/cloudpipe/cloudpickle/pull/276))
 
+- Internal refactoring of the pickling logic to make the code more predictable.
```

Comment:
Maybe be more specific an some how mention that this is about the detection of dynamic classes?